netjsonconfig/backends/airos/converters.py

```diff
@@ -0,0 +1,809 @@
+                'port': 514,
+                'status': 'disabled',
+            },
+            'status': 'enabled',
+        })
+        return (('syslog', result),)
+
+
+class System(AirOSConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+
+        result.append({
+            'airosx': {
+                'prov': {
+                    'status': 'enabled',
+                },
+            },
+            'cfg': {
+                'version': 0,
+            },
+            'date': {
+                'status': 'enabled',
+            },
+            'external': {
+                'reset': 'enabled',
+            },
+            'timezone': 'GMT',
+        })
+        return (('system', result),)
+
+
+class Telnetd(AirOSConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+
+        result.append({
+            'port': 23,
+            'status': 'disabled',
+        })
+        return (('telnetd', result),)
+
+
+class Tshaper(AirOSConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+
+        return (('tshaper', [{'status': 'disabled', }]),)
+
+
+class Unms(AirOSConverter):
+    netjson_keu = 'general'
+
+    def to_intermediate(self):
+
+        return (('unms', [{'status': 'disabled'}]),)
+
+
+class Update(AirOSConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+
+        result.append({
+            'check': {
+                'status': 'enabled',
+            },
+        })
+        return (('update', result),)
+
+
+class Users(AirOSConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+
+        result.append({
+            'status': 'disabled',
+        })
+
+        result.append([
+            {
+                'name': 'root',
+                'password': 'changeme',
+                'status': 'disabled',
+            },
+        ])
+
+        return (('users', result),)
+
+
+class Vlan(AirOSConverter):
+    netjson_key = 'interfaces'
+
+    def to_intermediate(self):
+        result = []
+        original = [
+                i for i in get_copy(self.netjson, self.netjson_key, []) if '.' in i['name']
+                ]
+
+        vlans = []
+        for v in original:
+            vlans.append({
+                'comment':  v.get('comment', ''),
+                'devname':  v['name'].split('.')[0],
+                'id':  v['name'].split('.')[1],
+                'status': status(v),
+            })
+
+        result.append(vlans)
+        result.append({
+            'status':  'enabled',
+        })
+
+        return (('vlan', result),)
+
+
+class Wireless(AirOSConverter):
+    netjson_key = 'interfaces'
+
+    def to_intermediate(self):
+        result = []
+        original = [
+                i for i in get_copy(self.netjson, self.netjson_key, []) if i['type'] == 'wireless'
+                ]
+
+        ws = []
```

Comment:
could you use something more readable like `wireless_interfaces` or `wireless_list`?